validator/packaging/policy.example

```diff
@@ -1,3 +1,18 @@
+# Copyright 2018 Intel Corporation
```

Comment:
It is not clear that we should be stamping example config files with copyright. Why would we do that? Isn't using example config files fair use?

Reply:
The parsing for this file does not directly handle comment characters. It may work by accident but comment characters are not described in the spec.

Reply:
removed the header here.